components/ActionSheet/src/MDCActionSheetController.m

```diff
@@ -255,11 +253,12 @@ - (void)updateTable {
 - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath *)indexPath {
   MDCActionSheetAction *action = self.actions[indexPath.row];
 
-  [self.presentingViewController dismissViewControllerAnimated:YES completion:^(void){
-    if (action.completionHandler) {
-      action.completionHandler(action);
-    }
-  }];
+  [self.presentingViewController dismissViewControllerAnimated:YES
```

Comment:
Since clang-format failed to leave it alone, I would expect that this did not meet the requirements for Google's Objective-C style guide.  

There is an exception carved-out to left-align the parameter (with a 4-space indent) and then align within the block from there. However, that only applies to invocations with "multiple inlined blocks". In this case we only have the one inlined block.

http://google.github.io/styleguide/objcguide.html#method-invocations